TemplePlus/python/python_object.cpp

```diff
@@ -48,6 +49,10 @@
 
 namespace py = pybind11;
 
+// Helper function for checking if information should be updated with chargen packet info
+bool UpdateWithChargenPacketInfo(objHndl handle) {
```

Comment:
Should add a comment on why this assumes IsSelectingFeats

Reply:
Thanks for the review.  I added the comment.  I'll try not to use hungarian in the future but I've been doing it for a long time...